src/player-statistics.js

```diff
@@ -80,8 +85,8 @@ export default [
   {group: 'Fractals of the Mists', title: 'Fractal Tonics', key: 'statistics.fractalTonics', sprite: 'endless-fractal-tonic', gold: false, by_playtime: true, decimals: 0},
   {group: 'Fractals of the Mists', title: 'Unstable Fractal Essence', key: 'statistics.unstableCosmicEssences', sprite: 'unstable-fractal-essence', gold: false, by_playtime: true, decimals: 0},
   {group: 'Strike Missions', title: 'Blue Prophet Crystals', key: 'statistics.blueProphetCrystal', sprite: 'blue-prophet-crystal', gold: false, by_playtime: true, decimals: 2},
-  {group: 'Strike Missions', title: 'Green Prophet Crystals', key: 'statistics.greenProphetCrystal', sprite: 'green-prophet-crystal', gold: false, by_playtime: true, decimals: 2},
-  {group: 'Strike Missions', title: 'Red Prophet Crystals', key: 'statistics.redProphetCrystal', sprite: 'red-prophet-crystal', gold: false, by_playtime: true, decimals: 2},
+  {group: 'Strike Missions', title: 'Blue Prophet Shards', key: 'statistics.blueProphetShard', sprite: 'blue-prophet-shard', gold: false, by_playtime: true, decimals: 0},
+  {group: 'Strike Missions', title: 'Green Prophet Shards', key: 'statistics.greenProphetShard', sprite: 'green-prophet-shard', gold: false, by_playtime: true, decimals: 0},
```

Comment:
Two things:
- Are we sure the crystals no longer exist? The wiki doesn't show them as discontinued.
- These keys don't exist, the account-statistics code has them with an `_` prefix, and then [aggregates them here](https://github.com/gw2efficiency/account-statistics/blob/master/src/statistics/aggregate.js#L101-L110)

Reply:
https://wiki.guildwars2.com/wiki/Blue_Prophet_Crystal

Blue Prophet Crystal exists

https://wiki.guildwars2.com/wiki/Red_Prophet_Crystal
Red Prophet Crystal no longer exists. (Red does not exist at all anymore in any form.)

https://wiki.guildwars2.com/wiki/Green_Prophet_Crystal
Green Prophet Crystals no long exist. (Only shards do.)

So existing items are: Blue shards and crystals and green shards.

This is because Anet changed their approach to this system. They wanted each strike era to have their own currency at first. But now there is only two, blue exists for old content and green exists for newest content. When there is a new expansion & its strikes arrive, the current green ones will also go into blue category.

They can change their mind in the future of course but right now this is how it works. 

I'll fix the keys, my bad.

Reply:
*Answered on Discord.*